### PR TITLE
build: update angular cli packages to version 12.0.0-next.9

### PR DESCRIPTION
**Pull request by alan-agius4**
This enables us to remove the workaround that we needed to consume `@angular-devkit/build-angular` from the snapshot builds.

### CODE REVIEW

**Comment by alan-agius4** [comment]
@AndrewKushnir & @petebacondarwin can you please review for size-tracking? Thanks.

File: goldens/size-tracking/integration-payloads.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
         "runtime-es2015": 3354,
         "main-es2015": 295076,
         "polyfills-es2015": 36975,
-        "434-es2015": 796
+        "src_app_lazy_lazy_module_ts-es2015": 825
```
Comment (alan-agius4) [inline goldens/size-tracking/integration-payloads.json:44]: Note: the size change is related to the fact that now the `sourceMappingURL` and chunk `id` have extra bytes due to the change in filename.

Example
```
(self["webpackChunkcli_hello_world_lazy"] = self["webpackChunkcli_hello_world_lazy"] || []).push([["src_app_lazy_lazy_module_ts"], {
```
```
//# sourceMappingURL=src_app_lazy_lazy_module_ts-es5.js.map
```

**Review by petebacondarwin — APPROVED** [review]
Reviewed-for: size-tracking

**Review by AndrewKushnir — APPROVED** [review]
Reviewed-for: size-tracking

**Review by josephperrott — APPROVED** [review]
LGTM

**Pull request closed without merging**